### PR TITLE
Branches: Disallow server version <5 for all branch commands

### PR DESCRIPTION
**Pull request by quinchs**

### CODE REVIEW

**Review by aljazerzen — APPROVED** [review]
Looks good.

File: src/commands/database.rs [hunk 1]
```diff
@@ -12,6 +12,10 @@ use crate::question;
 pub async fn create(cli: &mut Connection, options: &CreateDatabase, _: &Options)
     -> Result<(), anyhow::Error>
 {
+    if cli.get_version().await?.specific().major >= 5 {
+        anyhow::bail!("'edgedb database create' is deprecated in EdgeDB 5+. Please use 'edgedb branch create'");
```
Comment (aljazerzen) [inline src/commands/database.rs:16]: Oh, tests are failing because this is a hard error here.

You probably meant `eprintln!()`

Comment (quinchs) [reply src/commands/database.rs:16]: yeah I was gonna change lol

File: src/commands/database.rs [hunk 2]
```diff
@@ -23,6 +27,10 @@ pub async fn create(cli: &mut Connection, options: &CreateDatabase, _: &Options)
 pub async fn drop(cli: &mut Connection, options: &DropDatabase, _: &Options)
     -> Result<(), anyhow::Error>
 {
+    if cli.get_version().await?.specific().major >= 5 {
+        anyhow::bail!("'edgedb database drop' is deprecated in EdgeDB 5+. Please use 'edgedb branch drop'");
```
Comment (aljazerzen) [inline src/commands/database.rs:31]: ditto

File: src/commands/database.rs [hunk 3]
```diff
@@ -44,6 +52,10 @@ pub async fn drop(cli: &mut Connection, options: &DropDatabase, _: &Options)
 pub async fn wipe(cli: &mut Connection, options: &WipeDatabase, _: &Options)
     -> Result<(), anyhow::Error>
 {
+    if cli.get_version().await?.specific().major >= 5 {
+        anyhow::bail!("'edgedb database wipe' is deprecated in EdgeDB 5+. Please use 'edgedb branch wipe'");
```
Comment (aljazerzen) [inline src/commands/database.rs:56]: ditto

**Pull request merged**